.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,27 @@
+# Based on https://github.com/tk0miya/action-erblint/blob/main/README.md#example-usage
+
+name: Reviewdog ERB Lint
```

Comment:
We may eventually want to add a second job for running rubocop to this file. Can we keep the workflow name more general, like "Reviewdog"?